haskell/fizzbuzz.hs

```diff
@@ -0,0 +1,9 @@
+fizzbuzz :: [String]
+fizzbuzz = [if n `mod` 15 == 0 then "FizzBuzz" else
```

Comment:
``fizzbuzz = [if n `mod` 15 == 0 then "FizzBuzz" else``
FizzBuzz should show up if the number is a multiple of 5 & 3, not 15.